dev/reference/transactions/transaction-pessimistic.md

```diff
@@ -46,30 +50,12 @@ The `BEGIN PESSIMISTIC;` and `BEGIN OPTIMISTIC;` statements take precedence over
 
 To disable the pessimistic transaction mode, modify the configuration file and add `enable = false` to the `[pessimistic-txn]` category.
 
-## Configuration parameter
-
-The related configuration parameters are under the `[pessimistic-txn]` category. Besides `enable`, you can also configure the following parameters:
-
-- `ttl`
-
-    ```
-    ttl = "40s"
-    ```
-
-    `ttl` is the timeout for pessimistic transaction locking. Its default value is "40s" (40 seconds). You must set it to a value between 15~120 seconds, and a value out of this range can result in an error.
-
-    A transaction fails when its execution time exceeds `ttl`. If the value of `ttl` is set too high, the remaining pessimistic lock might block the write transaction for a long time when `tidb-server` is down. If set too low, the transaction might be rolled back by other transactions before it can finish execution.
-
-- `max-retry-count`
-
-    ```
-    max-retry-count = 256
-    ```
+## Difference with MySQL InnoDB
 
-    A pessimistic transaction can automatically retry a single statement. You can specify the maximum retrying times by setting the `max-retry-count` parameter to avoid retrying a statement endlessly in some extreme cases. Normally, you do not need to modify this configuration.
+1. TiDB uses range in the WHERE clause. When executing DML or `SELECT FOR UPDATE` statements, the concurrent `INSERT` statements within the range are not blocked.
```

Comment:
`When TiDB executing DML or `SELECT FOR UPDATE` statements that use range in the WHERE clause,`